text/0107-construct-library-module-lifecycle.md

```diff
@@ -30,20 +30,27 @@ module:
 - Stage 2 - Developer Preview
 - Stage 3 - GA
 
+Each stage in this document describes 3 distinct sets of activities:
+
+* **Entry activities:** Actions that need to be taken before the module enters this stage. These can either be automatic, or done by the team. They will usually be performed once a module is deemed for stage maturation, and all its exit criteria are met.
+* **Ongoing activities:** These are activities performed during the time a module lies statically in the given stage (e.g bug fixes, new features...)
+* **Exit criteria:** A module can progress to the next stage only if its exit criteria are met. Note that this is required, but not sufficient. A module matures to the next stage only if its exit criteria are met, **and** was prioritized over other modules using the [prioritization framework](#Construct-prioritization-framework).
```

Comment:
How about starting off with "When these criteria are all met, a module progresses into the next stage. Note that, while all of the criteria here are necessary, they may not be sufficient for a module to progress."?



> A module matures to the next stage only if its exit criteria are met, **and** was prioritized over other modules using the [prioritization framework](#Construct-prioritization-framework).

I'll probably drop this part. This is true for everything across this doc.

Further, if this is the only reason you've called 'necessary but not sufficient', probably better to drop that out as well.

Reply:
> I'll probably drop this part. This is true for everything across this doc

Yes, but this is a global section that relates to the entire doc as well. I think its fitting and actually this reasoning isn't stated anywhere else - is it? It also combines well with your comment here: https://github.com/aws/aws-cdk-rfcs/pull/133#discussion_r387101832

Reply:
I think this is okay to keep and is in the section that applies to the entire document. However, I think that we all agreed that they are prioritization tenets rather than a framework.

Reply:
Yeah, but lets push that change in a different PR. This isn't the only reference to it as a framework in the doc. I think its ok to leave it as is for now.